src/components/GeneralInfo/LoadingCard/LoadingCard.js

```diff
@@ -33,7 +33,14 @@ const valueToText = (value, singular, plural) => {
   return value || 'Not available';
 };
 
-export const Clickable = ({ value, target, plural, singular, onClick }) => {
+export const Clickable = ({
+  value,
+  target,
+  fullTarget,
```

Comment:
What's `fullTarget` for? Why do we need this?

Reply:
target basically specifies only what modal should be opened based on URL
thus if i want to redirect to a completely different page, i use a different parameter

Reply:
@xmicha82 Sebastian made a great suggestion, what if we pass in the link as `target` and then do a check in `LoadingCard` like so:

```to={target[0] === '/' ? target : `${pathname}/${target}`}```